.github/workflows/release_action.yaml

```diff
@@ -53,6 +52,9 @@ jobs:
         path: dist/
     - name: Publish distribution 📦 to PyPI
       uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
I don't think these are required, since the `pypi` environment is already configured for this job (the use of a short-lived API token eliminates the need for API tokens permanently configured in the secrets).

Reply:
Ah fair, I was following the PyBaMM release workflow and thought the addition would help with verification. Taking a second look at the [marketplace](https://github.com/marketplace/actions/pypi-publish#specifying-a-different-username) PyPi release action, they don't seem to be recommended anymore. Perhaps it should be removed from PyBaMM as well.

Reply:
Ah yes, we intend to remove those soon – it hasn't had a pair of eyes looking at it yet: https://github.com/pybamm-team/PyBaMM/pull/3358